README.md

```diff
@@ -163,3 +190,9 @@ docker exec -it openoni-dev bash
 ```bash
 ./docker-clean.sh
 ```
+
+**Remove persistent data containers and data volumes**
+```bash
+docker rm openoni-dev-data-mysql openoni-dev-data-solr
+docker volume rm $(docker volume ls -qf dangling=true)
```

Comment:
Do we want this to be automatic?  I could see people wanting to have dangling volumes so that they can switch between which volumes the persistent data is looking at.